packages/projects/src/constants.js

```diff
@@ -5,11 +5,14 @@ const components = {
   table: (props) => <table {...props} className="border-collapse table-auto w-full text-sm" />,
   th: (props) => <th {...props} className="border-b dark:border-slate-600 font-medium p-4 pl-8 pt-0 pb-3 text-slate-400 dark:text-slate-200 text-left" />,
   td: (props) => <td {...props} className="border-b border-slate-100 dark:border-slate-700 p-4 pl-8 text-slate-500 dark:text-slate-400" />,
-  h1: (props) => <h1 {...props} className="text-3xl" />,
-  h2: (props) => <h2 {...props} className="text-xl" />,
-  h3: (props) => <h3 {...props} className="text-lg" />,
-  ul: (props) => <ul {...props} className="list-disc" />,
-  ol: (props) => <ul {...props} className="list-decimal" />,
+  h1: (props) => <h1 {...props} className="text-4xl my-4" />,
+  h2: (props) => <h2 {...props} className="text-2xl my-2" />,
+  h3: (props) => <h3 {...props} className="text-xl my-1" />,
+  ul: (props) => <ul {...props} className="list-disc list-outside ml-4 mb-4" />,
+  ol: (props) => <ol {...props} className="list-decimal list-outside ml-4 mb-4" />,
+  li: (props) => <li {...props} className="my-2" />,
+  p: (props) => <p {...props} className="my-4" />,
```

Comment:
I did add the line break plugin so normal returns would work without having to specify <br/> but it wasn't working. It might be the order of running the plugins. Anyway, we should play with that and fix it or decide to move the plugin.

https://github.com/kernel-community/services/blob/main/packages/projects/package.json#L32

Reply:
@simonkernel: from what I can see, `remark-breaks` IS working. All it does is convert a single return to a `<br>`, which works for me in the create form when I inspect the output. If you remove `remarkBreaks` from the plugins and type:
```
foo
bar
```
it will appear as `foo bar`.

What's being addressed in the PR here is that _double_ returns convert to a `<p>` --which was already working-- but it's just that the `<p>` was completely unstyled, so it had no default top/bottom spacing, which we expect `<p>`s to have. That made the `<p>` look identical to a `<br/>`. That's what I fixed.

Reply:
Oh sweet. I must have just misunderstood its behavior. Good find! :1st_place_medal: